wiki/Beginner/en.md

```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
+The best way you can get practice for aiming without *osu!* is using your computer with the input device. Don't be afraid to practice by clicking around files on your computer if you're a mouse user, or to draw some art with your tablet. The added coordination will help you wonders later on!
 
 ### Speed
 
 Eventually, when you get to harder difficulties, you will need to click the notes more rapidly. Improving in this fact is actually quite simple; all you need to do is push yourself into clicking or tapping the keyboard faster. You can play games that require you to hit input as fast as possible, like Cookie Clicker or other button pushing games (they are very easy to find online if you just Google it).
 
 ### Practice
 
-Another heavy topic, is the fact that osu! is a game with an infinitely rising difficulty curve, meaning there is no "winning" the game. Instead, you'll be focused on improving yourself constantly. Instead of playing one map over and over, you will need to eventually move onto other maps once you start outclassing their difficulty.
+Another heavy topic, is the fact that *osu!* is a game with an infinitely rising difficulty curve, meaning there is no "winning" the game. Instead, you'll be focused on improving yourself constantly. Instead of playing one map over and over, you will need to eventually move onto other maps once you start outclassing their difficulty.
 
 The best way to improve is to challenge yourself. The only way you will get used to more and more tenacious difficulties is by exploring past your boundaries. Playing a map that makes you miss quite a few times grants ultimately more gameplay experience than playing many that you can get a full combo on or even get an SS rank on. This works much like an MMORPG, the harder the enemy, the more experience you will get for it.
 
-Don't be afraid to practise something that always makes you miss for one reason or another. You will need to find the challenge, and see what you need to do to meet that challenge. Do you need to hit the buttons faster? Do you need better timing? Do you need to be more precise when moving to notes? These are all questions (and more) that you can ask yourself. This will help you find your answer a lot more quickly and make your experience all the more worth it.
+Don't be afraid to practice something that always makes you miss for one reason or another. You will need to find the challenge, and see what you need to do to meet that challenge. Do you need to hit the buttons faster? Do you need better timing? Do you need to be more precise when moving to notes? These are all questions that you can ask yourself. This will help you find your answer a lot more quickly and make your experience all the more worth it.
```

Comment:
Practise is the British verb form of the word (as opposed to practice being the noun). Therefore, you don't need to alter them.